### PR TITLE
O share v3 unruin

### PR DESCRIPTION
**Pull request by adgad**
@gvonkoss Redid your changes (plus mobile fitting stuff) here

screen shot 2017-02-28 at 14 36 14
screen shot 2017-02-28 at 14 36 10
screen shot 2017-02-28 at 14 36 04

### CODE REVIEW

File: client/components/share/main.scss [hunk 1]
```diff
@@ -94,6 +84,10 @@ $o-icons-image-service-base-url: 'https://www.ft.com/__origami/service/image';
 
 		.article__save {
 			float: right;
+			button {
+				@include oButtonSize(bug);
```
Comment (gvonkoss) [inline client/components/share/main.scss]: should be big, elsewise there will be le bug

File: client/components/share/main.scss [hunk 2]
```diff
@@ -11,7 +11,6 @@ $o-icons-image-service-base-url: 'https://www.ft.com/__origami/service/image';
 		margin-top: 15px;
 		padding: 5px 0;
 		border-top: 1px solid getColor('warm-3');
```
Comment (gvonkoss) [inline client/components/share/main.scss]: it's the top border we want to get rid of I believe, maybe double check with @olipowell

File: client/components/share/main.scss [hunk 3]
```diff
@@ -94,6 +84,10 @@ $o-icons-image-service-base-url: 'https://www.ft.com/__origami/service/image';
 
 		.article__save {
 			float: right;
+			button {
+				@include oButtonSize(bug);
+				min-height: 42px;
```
Comment (gvonkoss) [inline client/components/share/main.scss]: as far as I've been told, @onishiweb has changed a few things in o-share, so this should be 40px

File: client/components/share/main.scss (hunk 2 above)
Comment (adgad) [inline client/components/share/main.scss]: Fixed

**Pull request merged**